README.md

```diff
@@ -32,7 +32,11 @@ module.exports = {
 ## Usage
 Use the only 3 apis for test cases:
 
-* `mockFlags(flags: LDFlagSet)`: mock flags at the start of each test case.
+* `mockFlags(flags: LDFlagSet, Options?: mockFlagsOptions)`: mock flags at the start of each test case. 
+###### Options
+| name | type | default | description |
+| ---- | ---- | ------- | ----------- |
+| skipFormatting(optional) | boolean | false | If true, flags will not be formatted to camelCase or kebab-case |
```

Comment:
I think this change can be removed now? It is no longer relevant with your change below.

```suggestion
`mockFlags(flags: LDFlagSet)`: mock flags at the start of each test case.
```

Reply:
Yes, sorry for missing this